SETUP.md

```diff
@@ -1,47 +1,44 @@
-# Install python somehow
+# Backend
 
-# Install virtualenv
+## Initial Setup
+
+#### Install python
+
+https://www.python.org/downloads/
+
+#### Install virtualenv
 
 https://docs.python-guide.org/dev/virtualenvs/
 https://stackoverflow.com/questions/31133050/virtualenv-command-not-found
 
-# Make virtual env and activate it
+#### Make virtual env and activate it
 
-virtualenv -p python3 kvasir-env
-source ~/kvasir/kvasir-env/bin/activate
+- Create: `virtualenv -p python3 kvasir-env`
+- Activate: `source ~/kvasir/kvasir-env/bin/activate`
 
-# Install dependencies
+#### Install dependencies
 
-pip install -r requirements.txt
+`pip install -r requirements.txt`
 
-# Dev db setup
+#### DB setup
 
 Install postgres locally (https://postgresapp.com/ is an easy way)
 
 Create db
-(in bash) > psql
-(in psql) > create database kvasir_dev;
-
-# Upgrade db
 
-> flask db upgrade
+- (in bash) > `psql`
+- (in psql) > `create database kvasir_dev;`
+- (in psql) > `create database kvasir_test;`
 
-# Test Setup
-
-Our integration tests use Firefox. In order to run tests, you'll need the geckodriver. Download [here](https://github.com/mozilla/geckodriver/releases).
-Once you've downloaded it, unzip the file and move it to `/usr/local/bin`: `mv ~/Downloads/geckodriver /usr/local/bin`.
-
-To run the integration tests, run `python integration-tests/sessions_test.py`.
-
-# TMDB API Key
+#### TMDB API Key
 
 Vist https://www.themoviedb.org/, make an account, and generate an API key. Add a `.env` file to the root, with contents of
 
 ```
 TMDB_API_KEY=YOUR_KEY
 ```
 
-# OMDB API Key
+#### OMDB API Key
```

Comment:
Should we just remove this since it's deprecated? Or is it not? Documentation below is unclear.